Kros.Utils/src/Kros.Utils/Properties/Resources.resx

```diff
@@ -117,79 +117,94 @@
   </data>
   <data name="Check_LessThan" xml:space="preserve">
-    <value>Hodnota parametra musí byť menšia ako {0}, ale skutočná hodnota je {1}.</value>
+    <value>Argument value must be less than {0}. Current value is {1}.</value>
   </data>
   <data name="Check_NotEmptyGuid" xml:space="preserve">
-    <value>Prázdny GUID nie je povolený.</value>
+    <value>Empty GUID is not allowed.</value>
   </data>
   <data name="Check_NotEqual" xml:space="preserve">
-    <value>Hodnota parametrov nesmie byť rovnaká {0}.</value>
+    <value>Argument value {0} is not allowed.</value>
   </data>
   <data name="Check_NotNull" xml:space="preserve">
-    <value>Hodnota nemôže byť null.</value>
+    <value>The value cannot be null.</value>
   </data>
   <data name="Check_StringNotNullOrEmpty" xml:space="preserve">
-    <value>Reťazec nemôže byť null, alebo prázdny.</value>
+    <value>The value cannot be null, neither empty string.</value>
   </data>
   <data name="Check_StringNotNullOrWhiteSpace" xml:space="preserve">
-    <value>Reťazec nemôže byť null, prázdny, ani zložený iba z bielych znakov.</value>
+    <value>The value cannot be null, nither empty string, nor string containing only whitespace characters.</value>
+  </data>
+  <data name="ColumnBelongsToAnotherIndex" xml:space="preserve">
+    <value>Column "{0}" cannot be added to index "{1}" because it already belongs to index "{2}".</value>
+  </data>
+  <data name="ColumnBelongsToAnotherTable" xml:space="preserve">
+    <value>Column "{0}" cannot be added to table "{1}" because it already belongs to table "{2}".</value>
+  </data>
+  <data name="ColumnMissingInBulkInsertDestinationTable" xml:space="preserve">
+    <value>Destination table "{0}" does not have column "{1}".</value>
+  </data>
+  <data name="ForeignKeyBelongsToAnotherTable" xml:space="preserve">
+    <value>Foreign key "{0}" cannot be added to table "{1}" because it already belongs to table "{2}".</value>
   </data>
-  <data name="DatabaseSchemaCache_UnsupportedDatabaseType" xml:space="preserve">
-    <value>Nepodporovaný typ pripojenia na databázu: {0}</value>
+  <data name="IBulkInsertFactoryNotRegisteredForClient" xml:space="preserve">
+    <value>{0} for ADO client "{1}" is not registered.</value>
   </data>
-  <data name="DatabaseSchemaLoader_UnsupportedConnectionType" xml:space="preserve">
-    <value>Nepodporovaný typ spojenia na databázu.</value>
+  <data name="IBulkInsertFactoryNotRegisteredForConnection" xml:space="preserve">
+    <value>{0} for connection type "{1}" is not registered.</value>
   </data>
-  <data name="IndexSchema_CannotChangeIndexNameWhenBelongsToTable" xml:space="preserve">
-    <value>Nie je možné zmeniť názov indexu, ak je v zozname indexov tabuľky. Tento index patrí tabuľke "{0}".</value>
+  <data name="IdGeneratorFactoryNotRegisteredForClient" xml:space="preserve">
+    <value>IdGeneratorFactory for ADO client {0} is not registered.</value>
```

Comment:
somewhere we have something in the sense of `class name is not registered ...` and elsewhere `{0} is not registered ...`. Can we unify it?